Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java

```diff
@@ -51,6 +51,36 @@ public View getView(final int position, View convertView, @NonNull ViewGroup par
             holder = (ViewHolder) view.getTag();
         }
 
+        if (position == 0) {
```

Comment:
These changes shouldn't be a part of your commit.

Reply:
ok